contribute_to_pim/contribution_guide.rst

```diff
@@ -390,7 +390,7 @@ messages of all the commits. When you are finished, execute the push command.
 .. _`coding standard`:                  http://docs.akeneo.com/latest/technical_architecture/best_practices/core/standards.html
 .. _`setup behat`:                      http://docs.akeneo.com/latest/technical_architecture/best_practices/core/behat.html
 .. _`Sylius documentation`:             https://github.com/Sylius/Sylius/tree/master/docs
-.. _`behat quick intro`:                http://docs.behat.org/en/v2.5/quick_intro.html
+.. _`behat quick intro`:                https://docs.behat.org/en/latest/quick_start.html
```

Comment:
Not sure putting latest is a good thing

Reply:
I agree, but there's no link with the version we are using.
So either with stay in version 2.5 (because the page exists), or we move to latest.